tests/Collection/CollectionFunctionalTest.php

```diff
@@ -698,32 +711,28 @@ function($collection, $session, $options = []) {
         ];
     }
 
-    public function collectionReadMethodClosures()
+    public function collectionReadMethodClosures(): array
     {
         return array_filter(
             $this->collectionMethodClosures(),
             function ($rw) {
-                if (strchr($rw[1], 'r') !== false) {
-                    return true;
-                }
+                return str_contains($rw[1], 'r');
```

Comment:
Small refactoring to use the PHP 8 function, provided by the polyfill for older versions.